app.rb

```diff
@@ -46,6 +46,34 @@
 Cuba.define do
   persist_session!
 
+  # We set the 'guests' layout as master template (file: /views/layout.guests.mote)
+  settings[:mote][:layout] = "layout.guests"
+
+  # Petition page
+  on "petition/:slug" do |slug|
```

Comment:
Yes, the campaigner will be able to see the `/petition/:slug` with no problems as now those paths (`/donation/:slug`, `/donate`) are inside `app.rb`, that makes these paths available for everyone.